unity/Assets/Scripts/AgentManager.cs

```diff
@@ -471,7 +484,9 @@ private enum serverTypes { WSGI, FIFO };
         float fieldOfView = DEFAULT_FOV,
         string skyboxColor = null,
         bool orthographic = false,
-        float? orthographicSize = null
+        float? orthographicSize = null,
+        float nearClippingPlane = 0.1f,
+        float farClippingPlane = 20.0f
```

Comment:
Instead of hardcoding these values can we instead grab them from this.primaryAgent.m_Camera.nearClipPlan/farClipPlane within the method if the passed in value is null?

Reply:
updated with the latest commit